addon/components/models-table.js

```diff
@@ -1022,10 +1023,6 @@ export default Component.extend({
     set(this, '_expandedRowIndexes', A([]));
   },
 
-  _setupSelectedRows() {
-    set(this, '_selectedItems', A([]));
```

Comment:
I wonder why they did it this way when columns are set directly instead of this setup step.  Maybe the assume you want to reset selected items every time it's loaded?